agent/src/softwarecontaineragentadaptor.h

```diff
@@ -0,0 +1,56 @@
+#ifndef SOFTWARECONTAINERAGENTADAPTOR_H
+#define SOFTWARECONTAINERAGENTADAPTOR_H
+
+#include "softwarecontaineragent.h"
+
+namespace softwarecontainer {
+
+class SoftwareContainerAgentAdaptor :
+    public com::pelagicore::SoftwareContainerAgent_adaptor
```

Comment:
One of the places where "pelagicore" is still used. Should it be changed here, or should I do it in my cleanup work?